### PR TITLE
test(e2e,playwright): Header

### PR DESCRIPTION
**Pull request by NeetigyaPod**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Related to #51705

### CODE REVIEW

**Comment by NeetigyaPod** [comment]
camperbot cant code

**Review by huyenltnguyen — COMMENTED** [review]
Hey @NeetigyaPod, I saw your message in Discord, but I'm commenting here so that other reviewers also have context.

In other to write tests for the case where `examInProgress` is `true`, we would want to go to the exam page and click "Start the exam", which would flip the `examInProgress` state to `true`.

You will need to wait for https://github.com/freeCodeCamp/freeCodeCamp/pull/51842 to be in before proceeding, since we need to populate the exams in the DB first.

**Comment by NeetigyaPod** [comment]
Makes sense, how is my test cases otherwise? Should I add/change something. This is my first contribution so asking.

File: e2e/header.spec.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+test('Has nav for universal', async ({ page }) => {
+  const universalNavigation = page.getByTestId(
+    headerComponentElements.universalNav
+  );
+  const universalNavigationLogo = page.getByTestId(
+    headerComponentElements.universalNavLogo
+  );
+  await expect(universalNavigation).toBeVisible();
+  await expect(universalNavigationLogo).toBeVisible();
```
[the remark this answers is not in the document]
Comment (NeetigyaPod) [reply e2e/header.spec.ts:32]: Added line

File: e2e/header.spec.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+});
+
+test.afterEach(async ({ page }) => {
+  await page.close();
+});
+
+test('Has link for skip content', async ({ page }) => {
+  const skipContent = page.getByTestId(headerComponentElements.skipContent);
+  await expect(skipContent).toBeVisible();
+});
```
[the remark this answers is not in the document]
Comment (NeetigyaPod) [reply e2e/header.spec.ts]: Added Line

**Comment by NeetigyaPod** [comment]
Hi, have made the changes as per your comments.

**Review by huyenltnguyen — COMMENTED** [review]
Thanks for the update!

You can actually start writing the remaining test cases with `examInProgress` being `true` by running `pnpm run seed:exams` in your development environment. CI will fail until #51842 is merged, but at least you're not blocked and have the tests ready for review by then.

**Comment by NeetigyaPod** [comment]
Hi, I have added the tests for the exam Navigation and Header testing is complete now.

**Review by huyenltnguyen — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**